js-functions_basic-functions/js/index.js

```diff
@@ -25,6 +29,10 @@ Then log all variables to the console again.
 */
 
 // --v-- write your code here --v--
+  console.log('Title :', bookTitle, 'Author :', author, 'Rating :', rating, 'Sales :', numberOfSales);
+rating = 5;
+numberOfSales = 38761734;
+console.log(logBookData());
```

Comment:
Your function logs to the console and don't return anything, so you shouldn't use console.log here. only: logBookData();